sunpy/coordinates/utils.py

```diff
@@ -420,7 +420,7 @@ def solar_angle_equivalency(observer):
     <Quantity 1.00603718 arcsec>
     """
 
-    if not isinstance(observer, (SkyCoord, BaseCoordinateFrame)):
+    if not isinstance(observer, SkyCoord | BaseCoordinateFrame):
```

Comment:
🥳

Reply:
Is this actually a good change? It seems amazingly pointless to me.

Reply:
I think its more readable, its much clearer that a logical operator is happening at that point imo

Reply:
THE RUFF SAYS WE DO IT SO WE DO IT